examples/polls-demo/app/cc/loading.tsx

```diff
@@ -1,5 +0,0 @@
-import { PollSkeleton } from "@/components/poll";
```

Comment:
I'll add this back - I temporarily removed it for testing with `useBackgroundQuery` where using a `Suspense` boundary directly was preferred

Reply:
Reminder to add this back :)